collectionstyles.css

```diff
@@ -11,22 +19,28 @@ div {
     display: flex;
     justify-content: center;
     flex-direction: column;
+    align-items: center;
+}
+
+.car {
+    padding: 5px;
 }
 
 
 img {
-    width: 150px;
+    width: 80%;
+    margin-top: 5px;
 }
 
```

Comment:
The images are now overlapping at lower resolutions
<img width="512" alt="image" src="https://user-images.githubusercontent.com/65457376/217859157-88074735-49ae-4706-86c0-b40e979ff022.png">